### PR TITLE
Add version dunder

### LINKED ISSUE #76: [BUG] No version dunder set for tsbootstrap
The following is failing:

```python
import tsbootstrap
tsbootstrap.__version__
```

However, sktime is using this to check if a compatible tsbootstrap version is installed.

### PR DESCRIPTION
**Pull request by benHeid**
Fixes #76

### CODE REVIEW

File: src/tsbootstrap/__init__.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+from importlib.metadata import version
+__version__ = version("tsbootstrap")
```
Comment (fkiraly) [inline src/tsbootstrap/__init__.py:2]: hm, why this way and not explicitly? Does this reduce the number of places to update?

Comment (benHeid) [reply src/tsbootstrap/__init__.py:2]: Yes I wanted to use only the information from the pyproject.toml file. However, if you prefer an explicit solution, I am happy to change it

Comment (fkiraly) [reply src/tsbootstrap/__init__.py:2]: I think this is better, if it works, as it reduces the number of update (and hence failure) points

Comment (fkiraly) [reply src/tsbootstrap/__init__.py:2]: should we do this in `sktime` too?

**Review by fkiraly — APPROVED** [review]

**Pull request merged**